crates/pathfinder/resources/invoke_contract_with_signature.json

```diff
@@ -0,0 +1,21 @@
+{
+    "entry_point_selector": "0x15d40a3d6ca2ac30f4031e42be28da9b056fef9bb7357ac5e85627ee876e5ad",
+    "signature": [
+        "3557065757165699682249469970267166698995647077461960906176449260016084767701",
+        "3202126414680946801789588986259466145787792017299869598314522555275920413944"
+    ],
+    "contract_address": "0x23371b227eaecd8e8920cd429357edddd2cd0f3fee6abaacca08d3ab82a7cdd",
+    "version": "0x0",
+    "max_fee": "0x4f388496839",
+    "calldata": [
+        "1",
+        "2925423296824367013529965983412166292018955438053992527907701360681569823649",
+        "1530486729947006463063166157847785599120665941190480211966374137237989315360",
+        "0",
+        "1",
+        "1",
+        "43",
+        "0"
+    ],
+    "type": "INVOKE_FUNCTION"
+}
```

Comment:
```suggestion
}
```

Reply:
Removing newlines from the end of JSON files is actually the default behavior of the VS Code JSON formatter.

I've added them manually.